domains.txt

```diff
@@ -200,14 +230,17 @@ email.welovesalt.com
 empiric.com
 empiricsolutions.co.uk
 emponics.co.uk
+emr.co.uk
 endeavourrecruitment.com
 energizerec.com
 energizerecruitment.co.uk
 energyjobline.com
 engagepsg.co.uk
 engagepsg.com
+englandassociates.co.uk
 enorthstaffing.com
 erg-uk.com
+erp-resources.co.uk
```

Comment:
Ever wondered why your Project Recruitment Costs are so high? So have we.

ERP Resources is a direct recruitment portal that connects ERP Professionals with Project and Programme Managers, Consultancies and End User clients across the ERP landscape in the UK. We are the first and only online recruitment solution that puts the contractor and the hirer in direct contact without going through an agency.